src/overlays/actors/ovl_Obj_Boyo/z_obj_boyo.c

```diff
@@ -1,4 +1,7 @@
 #include "z_obj_boyo.h"
```

Comment:
Sorry, 1 last comment! It's usually asked at the start of a PR, so sorry it's coming last!

So something to take in consideration that we are trying to get everybody who has contributed / wants to contribute to MM decomp to agree to licensing the repo before merging PRs from people who has not yet agreed to it. It is better explained here: https://github.com/louist103/mm-licensing/issues/1.
Could you please read about it and post in that issue if you agree or not. Thanks!